vm_insnhelper.c

```diff
@@ -934,7 +927,7 @@ vm_get_const_base(const VALUE *ep)
 
     while (cref) {
 	if (!CREF_PUSHED_BY_EVAL(cref) &&
-	    (klass = CREF_CLASS(cref)) != 0) {
```

Comment:
This check may be also removed if it is okay to remove the loop in `vm_get_cbase`.

Reply:
We still need a loop because some cref may be pushed by eval, but I have simplified the loop :)